simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -128,326 +127,330 @@
         "supportsBarcodeScanner": false,
         "supportsCardCreator": false,
         "supportsReservations": true,
-        "supportsSimplyESync": true
+        "supportsSimplyESync": true,
+        "uuid_id": "urn:uuid:4a490a70-0bd2-4f14-a81f-aeca4dda620b"
     },
     {
-        "authPasscodeAllowsLetters": true,
+        "authPasscodeAllowsLetters": false,
         "authPasscodeLength": 0,
         "catalogUrl": "https://bibliomation.lyrasistechnology.org/ansonia/",
-        "id": 30,
         "inProduction": false,
-        "legacy_id": 30,
         "logo": null,
-        "mainColor": "cyan",
+        "mainColor": "blue",
```

Comment:
I've verified that these changes correspond to what Ansonia's authentication document says.